### ISSUE TITLE
Proxmox install

### ISSUE THREAD

**Issue by babsenfred** [kippesikgithub/allsky#1]
Hello @kippesikgithub ,

I’m really interested in setting up an INDI-AllSky build with Proxmox. For the moment I have an AllSky camera (without INDI) and have some issues with blue images where there is no direct solution (other than switching from my RPI5 to RPI4) I’ve seen what INDI-Allsky can do and like to build a camera. I also have a Proxmox server running.  Because the howto for Proxmox is not online yet, I have a simple question. Can I install Ubuntu 24 as VM the normal way and after that installing INDI-Allsky like aaronwmorris wiki on Github?
Regards and thanks from Belgium :-)

**Comment by kippesikgithub** [comment]
Yeah, check the compatible version of Ubuntu or Debian. I'm running on Ubuntu 22.02 since the newer version wasnt compatible yet when i installed, and Ubuntu will help you since you not need to build the application, but just add it via ppa. Will finish the guide coming weeks
